dabl-meta.yaml

```diff
@@ -0,0 +1,17 @@
+catalog:
+    name: da-marketplace
+    version: 0.0.1
+    short_description: DA Marketplace
+    description: A marketplace for issuing and trading digital tokens.
+    release_date: 2020-09-30
+    author: Digital Asset (Switzerland) GmbH
+    url: https://github.com/digital-asset/da-marketplace
+    license: 0BSD
```

Comment:
Please add two tags: `dabl-sample-app` and `application`:

https://github.com/digital-asset/daml-dit-integration-slack/blob/master/dabl-meta.yaml#L13